devtools/x/src/changed_since.rs

```
@@ -15,15 +15,17 @@ pub struct Args {
 }
 
 pub fn run(args: Args, xctx: XContext) -> Result<()> {
-    let affected_set = changed_since_impl(&xctx, &args.base)?;
-    for package in affected_set.packages(DependencyDirection::Forward) {
-        println!("{}", package.name());
+    if let Some(_git_cli) = xctx.core().git_cli() {
+        let affected_set = changed_since_impl(&xctx, &args.base)?;
+        for package in affected_set.packages(DependencyDirection::Forward) {
+            println!("{}", package.name());
```

And here -- `changed-since` should fail if not available.